src/main/java/gov/cms/mat/cql_elm_translation/service/CqlParsingService.java

```diff
@@ -316,7 +335,9 @@ private ElementLookup buildElementLookupForValueSet(CQLValueSet cqlValueSet, Str
   }
 
   private Set<CQLDefinition> buildCqlDefinitions(CQLTools cqlTools) {
-    return cqlTools.getDefinitionContents().stream().map(this::buildCqlDefinition).collect(toSet());
+    Set<DefinitionContent> definitionContents = cqlTools.getDefinitionContents();
+    Map<String, Set<String>> usedDefinitions = cqlTools.getUsedDefinitions();
```

Comment:
looks like unused? or is this something that populates data?

Reply:
Good catch.. I must have had a copy/paste error